pipeline-examples/checkout-git-to-custom-directory/git-checout-custom-dir.groovy

```diff
@@ -0,0 +1 @@
+checkout scm: [$class: 'GitSCM', branches: [[name: '*/master']], userRemoteConfigs: [[url: 'https://github.com/jglick/simple-maven-project-with-tests']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'bla']]]
```

Comment:
`checkout` is misspelled in the filename.

As this is an example, there needs to be documentation in this file — what is this line doing? Which directory will it check out to? What makes this better than running `checkout scm` wrapped in `dir('bla') { … }`?

Wrapping over multiple lines would also be nice, rather than hiding the actual part you want to demonstrate at the veery end.